tests/misc/programs/misc/src/context.rs

```diff
@@ -213,16 +218,16 @@ pub struct TestInit<'info> {
 
 #[derive(Accounts)]
 pub struct TestInitZeroCopy<'info> {
-    #[account(init, payer = payer, space = 8 + size_of::<DataZeroCopy>())]
-    pub data: Loader<'info, DataZeroCopy>,
```

Comment:
We should keep `Loader` here since we need some minimal tests on it, even if deprecated.